app/src/main/java/uk/gov/onelogin/MainActivity.kt

```diff
@@ -5,6 +5,7 @@ import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
 import androidx.core.splashscreen.SplashScreen.Companion.installSplashScreen
 import androidx.navigation.compose.rememberNavController
+import uk.gov.ui.theme.GdsTheme
```

Comment:
How come we need this import? How is it used here?

Reply:
GdsTheme is used to skin the Material 3 components for the whole app